tap_adroll/client.py

```diff
@@ -72,9 +95,12 @@ def _make_request(self, method, endpoint, headers=None, params=None, data=None,
             params,
         )
 
+        if self.dev_mode :
+            headers = {}
+            headers['Authorization'] = 'Bearer %s' % self._access_token
```

Comment:
Does this mean that if you're not in dev mode you don't use an access token?

Reply:
If not in dev mode, we are using `OAuth2Session` which automatically fetches tokens and saves them. But, in dev mode, we are using the requests package (to prevent the automatic creation of tokens) where we have to explicitly add the existing `access_token` in the header.